src/compute/src/render/context.rs

```diff
@@ -406,11 +400,8 @@ where
         region: &Child<'a, S, S::Timestamp>,
     ) -> SpecializedArrangementImport<Child<'a, S, S::Timestamp>, T> {
         match self {
-            SpecializedArrangementImport::Bytes9Row(key_types, inner) => {
-                SpecializedArrangementImport::Bytes9Row(
-                    key_types.clone(),
-                    inner.enter_region(region),
-                )
+            SpecializedArrangementImport::RowUnit(inner) => {
+                SpecializedArrangementImport::RowUnit(inner.enter_region(region))
```

Comment:
Uncovered

Reply:
This should be hopefully covered now by a change in the test introduced in this PR. I focused on the cases unveiled by the coverage run, given that other paths in the PR were already covered by the existing tests.